.ops/build-docker/Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM ubuntu:22.04
+
+ARG USER_ID=1001
+ARG GROUP_ID=1001
+
+ENV DEFAULT_JAVA_VER 11
+ENV JAVA_HOME /usr/lib/jvm/java-${DEFAULT_JAVA_VER}-openjdk-arm64/
+ENV DEBIAN_FRONTEND noninteractive
+# ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV JDK_JAVA_OPTIONS "-Xmx512m -Xms512m -XX:MaxDirectMemorySize=700m -XX:ConcGCThreads=4 -XX:ParallelGCThreads=4"
+
+# tzdata ENVs required for installing software-properties-common
+ENV TZ=US/Eastern
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get -qq update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:openjdk-r/ppa && \
```

Comment:
does ubuntu 22.04 still need this repo to get java 11? that may just be the default ubuntu ships in its repos

Reply:
removed the repo